tests/test_parser.py

```diff
@@ -292,3 +292,25 @@ def test_implementation_constructor():
         entries2 = rispy.load(f, implementation=rispy.RisImplementation.WOK)
 
     assert entries1 == entries2
+
+
+def test_encodings():
+    fn = DATA_DIR / "example_utf_chars.ris"
+    p = Path(fn)
+
+    encoding1 = "ascii"
+    encoding2 = "utf-8"
+
+    with open(fn, "r", encoding=encoding2) as file:
+        expected = rispy.load(file)
+
+    try:
```

Comment:
use `pytest.raises` here https://docs.pytest.org/en/reorganize-docs/new-docs/user/pytest_raises.html

Reply:
thanks, didn't know this was a thing